src/app/api-connector/image.service.ts

```diff
@@ -234,15 +235,15 @@ export class ImageService {
 		const image_types: { [name: string]: Image[] } = {};
 		image_types[ImageTypes.IMAGE] = [];
 		image_types[ImageTypes.SNAPSHOT] = [];
-		image_types[ImageTypes.CUSTOM] = [];
+		image_types[ImageTypes.RESENV] = [];
 
 		for (const image of images) {
 			if (image.is_snapshot) {
 				image_types[ImageTypes.SNAPSHOT].push(image);
 			} else if (image.tags.includes('base_image')) {
 				image_types[ImageTypes.IMAGE].push(image);
-			} else {
-				image_types[ImageTypes.CUSTOM].push(image);
+			} else if (image.tags.filter(x => TemplateNames.ALL_TEMPLATE_NAMES.includes(x)).length > 0) {
```

Comment:
Should be ok for now

https://github.com/deNBI/cloud-portal-webapp/blob/fix/resenv_tab_filter/src/app/virtualmachines/conda/template-names.ts

https://github.com/deNBI/resenvs/blob/master/packer/tags.json

So "vscode" is missing , "cwlab" must be removed  -" jupyternotebook" must be changed to "jupyterlab" (but no  jupyterlab  pre-build-image is being built at the moment)

In the future it is better if we don't have hardcoded variant -> best request to API -> return "template_name" from all resenv models

You could do this for this PR, but for now it would be sufficient to adjust the values in ALL_TEMPLATES.